tornado-drivers/ptx/src/main/java/uk/ac/manchester/tornado/drivers/ptx/graal/lir/PTXArithmeticTool.java

```diff
@@ -357,11 +358,20 @@ public PTXLIROp genBinaryExpr(PTXBinaryOp op, LIRKind lirKind, Value x, Value y)
         return new PTXBinary.Expr(op, lirKind, x, y);
     }
 
+    private boolean isFastFMAEnabled() {
+        return TornadoOptions.ENABLE_FMA && TornadoOptions.FAST_MATH_OPTIMIZATIONS;
```

Comment:
Why is this condition &&?

Wouldn't it be correct to check only the `TornadoOptions.FAST_MATH_OPTIMIZATIONS`? and if it is not enabled then to check the `TornadoOptions.ENABLE_FMA` in order to emit `FMA`?

or must the programmer use both flags in order to emit `MAD`/`MAD_LO`?